### PR TITLE
Handle long query parameters

### PR DESCRIPTION
**Pull request by pakrym**
Large long values were falling into the float overload and having scientific format.

### CODE REVIEW

**Review by chamons — APPROVED** [review]

**Review by ShivangiReja — APPROVED** [review]

**Review by kasobol-msft — APPROVED** [review]

**Review by mikeharder — CHANGES_REQUESTED** [review]
I'm not sure if this is the correct fix.  Please don't merge until we have done more testing and investigation.

**Review by mikeharder — APPROVED** [review]
After more investigation I approve this fix.  All `double` values were falling into the `float` overload.  But only `double` values `>= 10,000,000` were being formatted with scientific notation.

**Pull request merged**
